AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_CubeMapCaptureAdded.py

```diff
@@ -18,23 +18,15 @@ class Tests:
     removal_undo = (
         "REDO CubeMap Capture Entity success",
         "P0: REDO CubeMap Capture Entity failed")
-
-    specular_ibl_property_set_to_very_high = (
+    specular_ibl_cubemap_quality_values = (
```

Comment:
is this tuple even being used now? with the for loop you are creating a dynamic f-string tuple in the for loop

Reply:
removed